Java/libraries/recognizers-text-sequence/src/main/java/com/microsoft/recognizers/text/sequence/spanish/SpanishPhoneNumberExtractorConfiguration.java

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.microsoft.recognizers.text.sequence.spanish;
```

Comment:
Do we need this? If I recall correctly, all non-CJK languages shoud fallback to the English PhoneNumber model as all formats should be supported in any language.

Reply:
We removed the `SpanishPhoneNumberExtractorConfiguration` as you mentioned.